src/components/organisms/settings/organization/createdOrganizationModal.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+import { useTranslation } from "react-i18next";
+
+import { ModalName } from "@enums/components";
+import { useModalStore } from "@src/store";
+
+import { Button } from "@components/atoms";
+import { Modal } from "@components/molecules";
+
+export const CreatedOrganizationModal = () => {
+	const { t } = useTranslation("settings", { keyPrefix: "organization" });
+	const { closeModal } = useModalStore();
+	const data = useModalStore((state) => state.data) as { orgName: string };
+
+	if (!data) return null;
+
+	return (
+		<Modal hideCloseButton name={ModalName.createdNewOrganization}>
+			<div className="mx-6">
+				<h3 className="mb-5 text-xl font-bold">{t("modal.organizationCreated", { name: data.orgName })}</h3>
```

Comment:
Please add on line 17:
```
const {orgName} = data;
```

Reply:
It's impossible, have ts error if `const {orgName} = data`